taskcluster/ci/tests/kind.yml

```diff
@@ -50,5 +50,5 @@ tasks:
             command: >-
                 pip install --user -r requirements/test.txt --require-hashes &&
                 pip install --user --no-deps . &&
-                pyenv local 3.10 3.9 3.8 3.7 3.6 &&
+                pyenv local 3.11 3.10 3.9 3.8 3.7 3.6 &&
```

Comment:
and here

Reply:
Yeah, I assume it's already removed from tox right? In that case no point in saving it. Would be nice to try and somehow share the list of supported versions better, but I don't have any clever suggestions